README.md

```diff
@@ -1,2 +1,47 @@
 # ProjectBadges
+
 Badges for Formidable Open Source Projects
+
+## Contents
+
+- [Using the package](#using-the-package)
+- [Component props](#component-props)
+  - [Examples](#examples)
+- [Try component locally](#try-the-component-locally)
+
+## Using the package
+
+- `yarn add formidable-oss-badges` or `npm install formidable-oss-badges`
+- In your react app, `import { ProjectBadge } from "formidable-oss-badges;`
+- Use the `<ProjectBadge />` component as directed below
+
+## Component props
+The `<ProjectBadge />` component takes five, optional, props:
+
+| Prop         | Type             | Description                                                                                  | Example             |
+| ------------ | ---------------- | -------------------------------------------------------------------------------------------- | ------------------- |
+| color        | String (hex)     | Base color of the badge                                                                      | `"#FF00FF"`         |
+| abbreviation | String           | Short representation of the name. Large font. Typically one uppercase letter, one lowercase. | `"Em"`              |
+| description  | String           | Title or brief description. Smaller text, displayed in all-caps.                             | `"Enzyme Matchers"` |
+| number       | Number or String | Number that appears at the top of the badge                                                  | `9` or `"09"`       |
+| className    | String           | Class to apply directly to the SVG                                                           | `"project-badge"`   |
+
+### Examples
+
+`<ProjectBadge color="#89E" abbreviation="No" description="Number"/>`
```

Comment:
It might be a bit nicer to use block code for this and the following code examples, e.g.

```jsx
<ProjectBadge color="#89E" abbreviation="No" description="Number"/>
```

so you can have syntax highlighting.